src/view/shell/index.web.tsx

```diff
@@ -16,11 +16,13 @@ import {useWebMediaQueries} from '../../lib/hooks/useWebMediaQueries'
 import {BottomBarWeb} from './bottom-bar/BottomBarWeb'
 import {useNavigation} from '@react-navigation/native'
 import {NavigationProp} from 'lib/routes/types'
+import {useAuxClick} from 'lib/hooks/useAuxClick.web'
```

Comment:
```suggestion
import {useAuxClick} from 'lib/hooks/useAuxClick'
```

The `.web` extension gets automatically added by the bundler